bundles/carts-rest-api/README.md

```diff
@@ -0,0 +1,10 @@
+# carts-rest-api
+[![license](https://img.shields.io/github/license/fond-of-kudu/product-image-storage-connector.svg)](https://packagist.org/packages/fond-of-kudu/product-image-storage-connector)
```

Comment:
should be 
[![license](https://img.shields.io/github/license/fond-of-kudu/carts-rest-api.svg)](https://packagist.org/packages/fond-of-kudu/carts-rest-api)